source/php-libraries.txt

```diff
@@ -160,3 +136,9 @@ Miscellaneous Projects
 - `PeclMongoPhpDoc <https://github.com/localgod/PeclMongoPhpDoc>`_ provides
   skeleton classes for the ``mongo`` extension, which may be used to support
   autocomplete and inline documentation for IDEs.
+
+- `PHPfastcache <https://github.com/PHPSocialNetwork/phpfastcache>`_ provides a simple
+high-performance backend cache system for MongoDB.
+
+- `XHgui <https://github.com/perftools/xhgui>`_ a Web interface for XHProf profiling data,
+storing it in MongoDB
```

Comment:
- Fixed indentation
- Fixed capitalization
- I think the rewording of the sentence explains the functionality of XHGui and XHProf a little better.

```suggestion
- `XHGui <https://github.com/perftools/xhgui>`_, a web interface for the XHProf profiler
  that stores profiling data in MongoDB.
```